content/en/synthetics/guide/explore-rum-through-synthetics.md

```diff
@@ -24,7 +24,7 @@ Synthetics Browser tests embed the RUM SDK, allowing you to explore:
 
 In your browser test recording, click **Collect RUM Data on** above the **Start Recording** button and select an application to collect data on. After saving your recording and test configuration, RUM gathers test data and generates session recordings from your browser test runs. 
 
-Optionally, you can choose the Default RUM Application displayed in the Browser Test Recorder:
+Optionally, you can choose which RUM application is displayed in the Browser Test Recorder:
```

Comment:
here there is the concept of default 
on the recorder page you have a RUM App selected https://share.getcloudapp.com/geumrvZk
but you can choose another one https://a.cl.ly/Z4u7wdJz

Any idea how we can reflect that?

Reply:
otherwise good to go for me

Reply:
```suggestion
Optionally, click `Synthetic Tests Default` and select a RUM application from the dropdown menu. Your application displays in the browser test recording:
```